react-app/src/components/DashboardPage/index.js

```diff
@@ -1,12 +1,87 @@
 import LeftNavigationBar from "../LeftNavigationBar";
-
+import { NavLink } from "react-router-dom";
+import { useSelector, useDispatch } from "react-redux";
+import { useEffect, useState } from "react";
+import {
+  getSettledExpenses,
+  getUnsettledExpenses,
+  getSummary,
+  getCreatedExpenses,
+  getCurrentExpense,
+} from "../../store/expense";
+import { fetchFriendships } from "../../store/friend";
+import MainHeader from "../MainHeader";
+import "./DashboardPage.css";
 function DashboardPage() {
-    return (
-        <>
-            <h1>Dashboard Page</h1>
-            <LeftNavigationBar />
-        </>
-    );
+  const dispatch = useDispatch();
+  const summary = useSelector((state) => state.expense.summary);
+  const friend =useSelector((state)=>(state.friend))
```

Comment:
notice how lines 18-20 are inconsistent with spacing around = and =>, and with the use of parentheses